docker/generate_dockerfiles.sh

```diff
@@ -92,7 +92,7 @@ function generate_main_dockerfile() {
                 conda_install='python=${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}
                                icu=58.1 libxml2 libxslt matplotlib mkl numpy paramiko
                                pandas psutil scikit-learn scipy traits=4.6.0' \
-                pip_install="grabbit==0.1.2 https://github.com/INCF/pybids/tarball/0.6.5" \
+                pip_install="pytest-xdist" \
```

Comment:
I removed the grabbit/pybids stuff, as it's not installed correctly to test `BIDSDataGrabber`, anyway. This should be fixed up in #2737.